### PR TITLE
Remove Jade from Travis description

### PR DESCRIPTION
**Pull request by Pandhariix**
Remove Jade from the CI section of the README

### CODE REVIEW

File: .travis.yml [hunk 1]
```diff
@@ -25,10 +25,8 @@ services:
 matrix:
   include:
     - env: DOCKER_VERSION=indigo-desktop-trusty
-    - env: DOCKER_VERSION=jade-desktop-trusty
     - env: DOCKER_VERSION=kinetic-desktop-xenial
     - env: DOCKER_VERSION=lunar-desktop-xenial
-    - env: DOCKER_VERSION=lunar-desktop-stretch
```
Comment (mikaelarguedas) [inline .travis.yml]: Unrelated change ?

Comment (Pandhariix) [reply .travis.yml]: Since it's not displayed in the README, there is no reason to keep it in the Travis Matrix

Comment (mikaelarguedas) [reply .travis.yml]: Stretch has proven to be a valuable CI platform as many system packages are different from Xenial (because Stretch is much more recent). I'd rather keep it. Feel free to add it to the README if that's the only motivation for removing it.

**Comment by nlyubova** [comment]
Maxime, Thank you for the fix!

**Comment by nlyubova** [comment]
Lgtm

**Comment by nlyubova** [comment]
And Thank you Mikael for reviewing!
